src/3d/qgs3dutils.cpp

```diff
@@ -82,6 +82,30 @@ AltitudeBinding Qgs3DUtils::altBindingFromString( const QString &str )
     return AltBindCentroid;
 }
 
+QString Qgs3DUtils::cullingModeToString( Qt3DRender::QCullFace::CullingMode mode )
+{
+  switch ( mode )
+  {
+    case Qt3DRender::QCullFace::NoCulling: return QStringLiteral( "no-culling" );
+    case Qt3DRender::QCullFace::Front: return QStringLiteral( "front" );
+    case Qt3DRender::QCullFace::Back: return QStringLiteral( "back" );
+    case Qt3DRender::QCullFace::FrontAndBack: return QStringLiteral( "front-and-back" );
+    default: return QString();
```

Comment:
Can we drop this default and return QString() outside the switch? I think we should avoid using defaults wherever possible so that we get the compiler warnings when the enum changes.